CONTRIBUTING.md

```diff
@@ -47,8 +47,11 @@ terraform plan
 ```
 
 If you are using a local build of the P0 API server, you can also set that in your
-environment:
+`main.tf`:
 
-```bash
-export P0_HOST=...
+```
+provider "p0" {
+  org = "p0-nathan"
```

Comment:
Nit: Do we want to use an example that is more customer-relevant, e.g "customer1" or "demo"? (Not necessarily great examples either.)

Reply:
I think this should be more developer-relevant since it's for our `CONTRIBUTING.md` file which devs are reading.